nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/GetHTTP.java

```diff
@@ -197,6 +198,30 @@
             .addValidator(StandardValidators.PORT_VALIDATOR)
             .build();
 
+    public static final String DEFAULT_COOKIE_POLICY_STR = "default";
+    public static final String STANDARD_COOKIE_POLICY_STR = "standard";
+    public static final String STRICT_COOKIE_POLICY_STR = "strict";
+    public static final String NETSCAPE_COOKIE_POLICY_STR = "netscape";
+    public static final String IGNORE_COOKIE_POLICY_STR = "ignore";
+    public static final AllowableValue DEFAULT_COOKIE_POLICY = new AllowableValue(DEFAULT_COOKIE_POLICY_STR, DEFAULT_COOKIE_POLICY_STR,
+            "Default cookie policy that provides a higher degree of compatibility with common cookie management of popular HTTP agents for non-standard (Netscape style) cookies.");
+    public static final AllowableValue STANDARD_COOKIE_POLICY = new AllowableValue(STANDARD_COOKIE_POLICY_STR, STANDARD_COOKIE_POLICY_STR,
+            "RFC 6265 compliant cookie policy (interoperability profile).");
+    public static final AllowableValue STRICT_COOKIE_POLICY = new AllowableValue(STRICT_COOKIE_POLICY_STR, STRICT_COOKIE_POLICY_STR,
+            "RFC 6265 compliant cookie policy (strict profile).");
+    public static final AllowableValue NETSCAPE_COOKIE_POLICY = new AllowableValue(NETSCAPE_COOKIE_POLICY_STR, NETSCAPE_COOKIE_POLICY_STR,
+            "Netscape draft compliant cookie policy.");
+    public static final AllowableValue IGNORE_COOKIE_POLICY = new AllowableValue(IGNORE_COOKIE_POLICY_STR, IGNORE_COOKIE_POLICY_STR,
+            "A cookie policy that ignores cookies.");
+
+    public static final PropertyDescriptor REDIRECT_COOKIE_POLICY = new PropertyDescriptor.Builder()
+            .name("redirect-cookie-policy")
+            .displayName("Redirect Cookie Policy")
+            .description("When a HTTP server responds to a request with a redirect, this is the cookie policy used to copy cookies to the following request.")
+            .allowableValues(DEFAULT_COOKIE_POLICY, STANDARD_COOKIE_POLICY, STRICT_COOKIE_POLICY, NETSCAPE_COOKIE_POLICY, IGNORE_COOKIE_POLICY)
+            .defaultValue(DEFAULT_COOKIE_POLICY_STR)
```

Comment:
I was just about to merge this in, and realized that we might want to have a different default for 0.x and 1.x. 0.x prior to this had CookieSpecs.STANDARD set, and this would potentially change behavior without reconfiguring a flow back to standard, but admittedly, I had no success making a test that worked with CookieSpecs.STANDARD and broke with CookieSpecs.DEFAULT. @mosermw  - how do you feel about .defaultValue(STANDARD_COOKIE_POLICY_STR) in 0.x for flow compatibility versus writing a migration note, and .defaultValue(DEFAULT_COOKIE_POLICY_STR) in 1.x which is what I think a reasonable default where flow compatibility is less an issue?